include/revng/PipelineC/Tracing/Trace.h

```diff
@@ -124,6 +124,9 @@ struct RunTraceOptions {
   std::set<uint64_t> BreakAt;
   // If set, all temporary directories will be created within this directory
   std::string TemporaryRoot;
+  // Instead of using a temporary directory, the first invocations will use
+  // these directories instead
+  std::vector<std::string> ResumeDirectories;
```

Comment:
The fact that you can specify multiple resume directories (one for the first manager being created, one for the second) seems quite subtle.

I don't think anyone will use it, in practice. I'd go for the simpler "override all resume directories option", or we can assert if you specified the override directory and more than one manager is created.

Reply:
The "override all resume directories option" could be very dangerous (think 2 managers running simultaneously)
I'll implement the second variant where it'll `revng_abort` if a second manager is created